android_docker_image.md

```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
```

Comment:
Rephrase this to something that a stranger to Semaphore can understand.

Eg. Configuring your Android project on Semaphore